src/ScaleUnitManagement/UserConfig.sample.xml

```diff
@@ -23,6 +23,12 @@
     <!-- If empty, the existing value in web.config will be used - otherwise the web.config will get overwritten with this value. -->
     <AADTenantId></AADTenantId>
   </AADConfiguration>
+
+  <ADFSConfiguration>
```

Comment:
Hm, these fields exist already on the AADConfiguration section. I would prefer to avoid adding more configuration details here. Can we reuse the AADConfiguration section for this? Maybe change the AADConfiguration to something else (maybe AuthConfiguration)?

Reply:
We can reuse the same section. However, we still need to use different App to call into Hub and different one to call into LBD.
That means we still need to specify AppId, AppSecret and Authority twice.

Another way to implement this could be to have something like this:

<AuthConfiguration>
       <AADConfiguration>
                 <appid>
                  <authority> ...
       <ADFSConfiguration>
                 <appid>
                  <authority> ...

Reply:
Or we can use some different naming ...

Reply:
So basically what you say is that now the auth details are not global, instead they depend on the type of each scale unit. Should we then move this info down to be part of the scale unit instance?

Assuming we have 10 LBD scale units, would it be the same secret/appid/authority used on all of them?

Reply:
That sounds like a good plan @alkalomo (moving the ADFS settings into the scale unit configuration): it's a fair assumption to make that each LBD scale unit will have their own ADFS (as they are inherently entirely offline).

Reply:
Yes, they are now specific for scale unit. So if we have 10 LBD scale units, we will have 10 apps. Unless they are configured against same ADFS which I think is also possible. @AndreasHassing probably knows more about that.

Reply:
So should we move then ADFS config in ScaleUnitInstance node. But only to the LBD one. In case of other scale unit, it can be empty ?
And then leave global AADConfig as it is ?

Reply:
I updated UserConfig template. Let me know if the current configuration is okay.